### PR TITLE
Blog post on resetting stats with production switch

### PR DESCRIPTION
**Pull request by orta**
:100:

### CODE REVIEW

File: _posts/2015-08-09-Stats-Reset.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+categories: cocoapods website
+---
+
+TL;DR: _CocoaPods Stats is ready to move from it's current staging environment_, and we will be resetting all stats next weekend.
+
+<!-- more -->
+
+When I set up CocoaPods Stats, the API + analytics database with the help from [Segment](http://segment.com). I created it within a staging environment in our analytics database. I gave it two months to gather enough data in order for us to be sure that the at-the-time evolving table data structure works. 
+
+Looks like it is working. In that time, people using only the latest releases of CocoaPods, have used CocoaPods to grab [AFNetworking](https://cocoapods.org/pods/AFNetworking) almost 100,000+ times.
```
Comment (segiddins) [inline _posts/2015-08-09-Stats-Reset.markdown]: clarify that's downloads, versus the number of apps that are using it?

File: _posts/2015-08-09-Stats-Reset.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+So, then next week I will migrate the stats server to production and everything will be back to zero downloads / installs again.
```
Comment (segiddins) [inline _posts/2015-08-09-Stats-Reset.markdown]: ~~then~~

**Comment by orta** [comment]
agreed on both accounts :+1:

**Comment by segiddins** [comment]
:+1:

File: _posts/2015-08-09-Stats-Reset.markdown [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+title:  "CocoaPods Stats Reseting"
+author: orta
+categories: cocoapods website
+---
+
+TL;DR: _CocoaPods Stats is ready to move from it's current staging environment_, and we will be resetting all stats next weekend.
+
+<!-- more -->
+
+When I set up CocoaPods Stats, the API + analytics database with the help from [Segment](http://segment.com). I created it within a staging environment in our analytics database. I gave it two months to gather enough data in order for us to be sure that the at-the-time evolving table data structure works.
```
Comment (AliSoftware) [inline _posts/2015-08-09-Stats-Reset.markdown]: …with the help from Segment*, I created it…

(Otherwise the sentence is missing a verb ;-))

Comment (orta) [inline _posts/2015-08-09-Stats-Reset.markdown]: damn you foreigners and your good englishness

Comment (orta) [inline _posts/2015-08-09-Stats-Reset.markdown]: Fixed in  https://github.com/CocoaPods/blog.cocoapods.org/commit/c7df06a2d2c774159b456ce5a447f3457854fb97

Comment (AliSoftware) [inline _posts/2015-08-09-Stats-Reset.markdown]: 🍷

**Pull request merged**